server.py

```diff
@@ -8,7 +9,35 @@ def desk():
     return render_template("index.html")
 
 
-# def ask(word):#
+@app.route("/tagged/word/%string word%")
+def req():
+    # return tagged word#
+    data = request.args.get('word', {}, type=str)
+    return jsonify(dict(results=str(data)))
+
+
+@app.route("/tagged/sentence/")
+def tag_sent():

```

Comment:
It will be better to use present simple:
```suggestion
def send_tag():
```
Do the same with names of all functions.

Reply:
It is not about sending smth, just tagged sentence

Reply:
Oh, sorry.